src/summaries/blockchain_basics.yaml

```diff
@@ -1,12 +1,67 @@
 - title: Introduction to Blockchain
   key: introduction-to-blockchain
   details: |
-    - Persistence: once added, data can never be removed*
-    - Consensus: all honest participants have the same data**
-    - Liveness: honest participants can add new transactions
-    - Open(?): anyone can add data
+    - What is Blockhain ?
```

Comment:
We can create three summaries from this since one
1) Into to Blockchain (What is Blockhain & Early days of blockchain)
2) Working of Blockchain (Blockchain Ledger & Blockchain Transactions and Blocks )
3) Types of Blockchain Networks (Public, private .....)